From 20bbe559a43587545a8f1f2ac3ea98b687482efd Mon Sep 17 00:00:00 2001
From: Jun Tian <jun.j.tian@intel.com>
Date: Thu, 2 Aug 2012 11:30:52 +0800
Subject: Enable sound effect support in the SDK system image

Add the missing configuration in sdk.mk to support sound effect in the SDK building.
This patch can fix the CTS sound effect failues of media test in emulator system image.

Change-Id: I7dba37d067910eb40acc3e7f2085cc276ac84800
---
 target/product/sdk.mk |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/target/product/sdk.mk b/target/product/sdk.mk
index 51b90f5..c56d445 100644
--- a/target/product/sdk.mk
+++ b/target/product/sdk.mk
@@ -84,11 +84,12 @@ PRODUCT_COPY_FILES := \
 	device/generic/goldfish/camera/media_codecs.xml:system/etc/media_codecs.xml \
 	frameworks/native/data/etc/android.hardware.touchscreen.multitouch.jazzhand.xml:system/etc/permissions/android.hardware.touchscreen.multitouch.jazzhand.xml \
 	frameworks/native/data/etc/android.hardware.camera.autofocus.xml:system/etc/permissions/android.hardware.camera.autofocus.xml \
+	hardware/libhardware_legacy/audio/audio_policy.conf:system/etc/audio_policy.conf \
 	frameworks/av/media/libeffects/data/audio_effects.conf:system/etc/audio_effects.conf \
-	hardware/libhardware_legacy/audio/audio_policy.conf:system/etc/audio_policy.conf
 
 include $(SRC_TARGET_DIR)/product/emulator.mk
 
+$(call inherit-product-if-exists, frameworks/base/data/sounds/AllAudio.mk)
 $(call inherit-product-if-exists, frameworks/base/data/fonts/fonts.mk)
 $(call inherit-product-if-exists, frameworks/base/data/keyboards/keyboards.mk)
 $(call inherit-product-if-exists, frameworks/webview/chromium/chromium.mk)
-- 
1.7.4.1

